alf/algorithms/ddpg_algorithm.py

```diff
@@ -355,6 +358,11 @@ def calc_loss(self, info: DdpgInfo):
 
         actor_loss = info.actor_loss
 
+        if self._critic_losses[0]._improve_w_nstep_bootstrap:
```

Comment:
This masking seems quite hacky. CriticLoss should be responsible for actor losses or other losses in general. So either this flag really belongs to an algorithm or some other places.